packages/devextreme/js/renovation/component_wrapper/editors/editor.ts

```diff
@@ -157,7 +157,7 @@ export default class Editor extends Component {
     super._optionChanged(option);
   }
 
-  reset(): void {
+  clear(): void {
```

Comment:
we should make this change in editors.js too to keep the two versions (js and renovated) syncronized